requirements.txt

```diff
@@ -1,18 +1,17 @@
 jsonschema==2.6.0
 requests==2.20.0
 jsonrpcserver==3.5.6
+jsonrpcclient==3.3.5
 sanic==18.12.0
 gunicorn==19.9.0
 grpcio==1.25.0
 grpcio-tools==1.25.0
 protobuf==3.7.0
 aiohttp==3.4.4
-jsonrpcclient==2.6.0
 secp256k1==0.13.2
 sanic-cors==0.9.6
 earlgrey>=0.0.4
 iconcommons>=1.0.5
 pytest
 pytest-asyncio
-asyncmock
-pytest-mock
+mock
```

Comment:
please specify mock version and move to above of `pytest`.

Reply:
Thank you.
This is the major problem that i suffered on travis-ci test failure.

I set mock version above 4.0.1 which is latest.